src/Components/Row.js

```diff
@@ -86,13 +86,23 @@ class Row extends Component<Props> {
 export default withStyles((theme) => ({
   root: {
     width: "100%",
+    maxWidth: (props) => props.width,
+    margin: "0 auto",
     height: "auto",
     overflowY: "hidden",
     overflowX: "scroll",
-    padding: "8px",
+    padding: (props) =>
+      `${props.paddingTop}px ${props.paddingRight}px ${props.paddingBottom}px ${
+        props.paddingLeft
+      }px`,
+    background: (props) =>
```

Comment:
Can we break these out into separate properties?

`backgroundImage`
`backgroundSize`
`backgroundPosition`
`backgroundRepeat`